server/handlers/image_handler.py

```diff
@@ -6,14 +6,15 @@ class ImageHandler:
     def __init__(self, gs):
         self.gs = gs
 
+    # When socket connection is not used
```

Comment:
So dummy? If so you may want to comment that in.

Reply:
The point of the method is to pretend to be a socket connection, and call `process_image` at about every 2 seconds to simulate data traffic. If a socket connection is being used, then `process_image` is being called directly. I've decided on just using one class, but adding a method to be used for this scenario.